ecsmanage/management/commands/ecsmanage.py

```diff
@@ -164,40 +166,37 @@ def run_task(self, config, task_def_arn, security_group_id, subnet_id, cmd):
         Run a task for a given task definition ARN using the given security
         group and subnets, and return the task ID.
         """
-        overrides = {"containerOverrides": [{"name": "django", "command": cmd}]}
```

Comment:
This variable wasn't being in multiple places, so I opted to include it in the `kwargs` dictionary structure.